docs/dev/python/table-api-users-guide/table_environment.md

```diff
@@ -0,0 +1,838 @@
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>from_pandas(pdf, schema=None, split_num=1)</strong>
+      </td>
+      <td>
+        Creates a table from a pandas DataFrame. 
+      </td>
+      <td class="text-center">
+        <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.TableEnvironment.from_pandas">link</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>from_path(path)</strong>
+      </td>
+      <td>
+        Reads a registered table from catalog and returns the resulting `Table` object. 
+      </td>
+      <td class="text-center">
+        <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.TableEnvironment.from_path">link</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>sql_query(query)</strong>
+      </td>
+      <td>
+        Evaluates a SQL query on registered tables and retrieves the result as a `Table` object. 
+      </td>
+      <td class="text-center">
+        <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.TableEnvironment.sql_query">link</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>create_temporary_view(view_path, table)</strong>
+      </td>
+      <td>
+        Registers a `Table` object as a temporary view similar to SQL temporary views. 
+      </td>
+      <td class="text-center">
+        <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.TableEnvironment.create_temporary_view">link</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>drop_temporary_view(view_path)</strong>
+      </td>
+      <td>
+        Drops a temporary view registered in the given path. 
+      </td>
+      <td class="text-center">
+        <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.TableEnvironment.drop_temporary_view">link</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>drop_temporary_table(table_path)</strong>
+      </td>
+      <td>
+        Drops a temporary table registered in the given path. 
+        You can use this interface to drop the temporary source table and temporary sink table.
+      </td>
+      <td class="text-center">
+        <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.TableEnvironment.drop_temporary_table">link</a>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>execute_sql(stmt)</strong>
+      </td>
+      <td>
+        Execute the given single statement, and return the execution result.
+        The statement can be DDL/DML/DQL/SHOW/DESCRIBE/EXPLAIN/USE.
+        Note that for "INSERT INTO" statement this is an async operation.
+        This is usually expected when submitting a job to a remote cluster. However, when executing a job in a mini cluster or IDE, you need to wait until the job execution finished, then you can refer to the following code: <br>
```

Comment:
Add a reference to https://ci.apache.org/projects/flink/flink-docs-release-1.11/dev/python/faq.html#wait-for-jobs-to-finish-when-executing-jobs-in-mini-cluster? It has detailed explanation about this.